assets/js/plugin-check-admin.js

```diff
@@ -129,19 +132,33 @@
 	/**
 	 * Run Checks.
 	 *
+	 * @since n.e.x.t
+	 *
 	 * @param {Object} data The response data.
+	 */
+	async function runChecks( data ) {
+		for ( let i = 0; i < data.checks.length; i++ ) {
+			const result = await runCheck( data.plugin, data.checks[ i ] );
+			console.log( result );
```

Comment:
```suggestion
            try {
			   const result = await runCheck( data.plugin, data.checks[ i ] );
			   console.log( result );
			} catch (e) {
              // Ignore for now.
             }   
```

My point about error handling seems to have been missed.

Reply:
Thanks @spacedmonkey this has been updated.